marlowe-dashboard-client/src/Data/Bimap.purs

```diff
@@ -0,0 +1,143 @@
+module Data.Bimap
```

Comment:
I would love to open source it eventually. Same with `Halogen.Form`. I think for now, it makes sense to "incubate" these in the Marlowe Run codebase before publishing them - they aren't complete yet and for a while we will benefit from the ability to rapidly iterate on it.

Reply:
I was able to put this together so quickly as well because it's more or less a copy of an existing Haskell lib https://hackage.haskell.org/package/bimap-0.4.0